### ISSUE TITLE
Use alt instead of aria-label for images in adapt-contrib-graphic

### ISSUE THREAD

**Issue by amygroshek** [adaptlearning/adapt_framework#2050]
### Default to alt tag instead of aria-label for images in graphic component
Right now the graphic component is assigning the `alt` value entered in the component JSON as an `aria-label` tag. In most cases this will work fine with screen readers, but there are some failures, and accessibility testing tools may complain. 

### Your environment
* 2.0.6
* all browsers
* Macbook Pro 2012, MacOS High Sierra

### Steps to reproduce
Create a graphic component and `grunt build`. View the component and inspect source. 

### Expected behaviour
Image should have `alt` tag with alt value entered for component

### Actual behaviour
Image has `aria-label` attribute with alt value entered for component

Have created pull request [#65](https://github.com/adaptlearning/adapt-contrib-graphic/pull/65) to address.

**Comment by oliverfoster** [comment]
We shouldn't do this for the reason listed here [HTML Accessibility API Mappings 1.0](https://w3c.github.io/html-aam/#img-element).
Need to document this for public consumption in line with headings, lists and other html semantic elements. We shouldn't use older tags/attributes when the aria spec supersedes them even when the WCAG tests say otherwise.

**Comment by oliverfoster** [comment]
decidedly fine as it is.
